test/server/approval.test.js

```diff
@@ -73,7 +73,7 @@ describe('Approval#countApprovalsAndVetos', () => {
       body: ':+1:' // is ignored because mfellner already approved
     }]
     try {
-      const {approvals} = await Approval.countApprovalsAndVetos(null, DEFAULT_REPO, comments, DEFAULT_CONFIG.approvals, null)
+      const {approvals} = await (new Approval(null, null)).countApprovalsAndVetos(DEFAULT_REPO, comments, DEFAULT_CONFIG.approvals)
```

Comment:
Let's instantiate the Approval with it's own variable on the line above to make things a bit cleaner.